src/Versionable.php

```diff
@@ -23,6 +23,18 @@ trait Versionable
 
     public static function bootVersionable()
     {
+        if (config('versionable.create_initial_versions')) {
+            static::updating(
```

Comment:
created?

Reply:
I believe `created` event would not be fired on already created models?
This pr is intended to keep at least the existing/current state of model.

Reply:
I believe `created` event would not be fire on already created Models?
The PR intends to keep at least the existing/current state of the Model.

Thanks.

Reply:
@overtrue Please let me if you need me to change anything. 

I am working on a plugin that wraps this package. Would be great if we could get this pr in.

Hope you are having a great weekend.
Cheers!

Reply:
我认为，对已经有的数据，可以用一个命令（比如：`./artisan version:init --model=App\Post`），或者其他方式来实现，我认为 `create_initial_versions` 应该是 “模型创建时也同时创建版本” 的直观理解。

Reply:
I also thought that the command should be enough, but sometimes we could forget running the command and end up losing a version.

Right now, this is the solution on top of my head.

We can discuss more ways of doing it. I am happy to try something more ideal.

Reply:
Yes, my concern is that some people don't intuitively understand that the initial version is created on update. As I said above, `create_initial_versions` should be understood as `create an initial version on model creation', which sums up to the fact that this configuration doesn't match the behavior. `create_version_if_not_exists` might match your expectation better, but it will cause a conflict of understanding: "create init version on updated" or "create init version both created and updated"?  What do you think?

Reply:
Oh yes, i get your point now.

Naming things is hard man LOL.

`create_version_if_not_exists` sounds good. Or may be something like: 

* `keep_existing_version`
* `keep_original_version`
* `keep_current_version`

Reply:
keep_original_version +1

Reply:
And I would like to change `static::updating` to `static::saved`, What do you think?

Reply:
Hmm, i think saved would execute on both updated and created events?
Let me double check.

Reply:
yes

Reply:
Plus saved method would too late to create the initial version. At the time of `updating` model event, the changes would not be persisted into DB and we can still get the original version of Model.

Reply:
Hmm, right, so let's do that, keep `updating`, but I'd be worried that if the user sets `keep_original_version` to true, but doesn't find any version after the model is created, they'll think it's a bug, so listen for both created and updating?

Reply:
I think it already creates a version even when you create for first time.

Reply:
> Hmm, right, so let's do that, keep `updating`, but I'd be worried that if the user sets `keep_original_version` to true, but doesn't find any version after the model is created, they'll think it's a bug, so listen for both created and updating?

Confirmed. It already creates the version when you create a row for first time.

Reply:
> > Hmm, right, so let's do that, keep `updating`, but I'd be worried that if the user sets `keep_original_version` to true, but doesn't find any version after the model is created, they'll think it's a bug, so listen for both created and updating?
> 
> Confirmed. It already creates the version when you create a row for first time.

👌